### DISCUSSION TITLE
Bagaimana menulis variabel di javascript?

### DISCUSSION THREAD

**Discussion by Daezyyy** [Daezyyy/javascripttttt#9] [Q&A] [answered]
lets strart discussions!

**Comment by DillonAdriyan** [comment, marked as answer]
### Variabel:
Mendeklarasikan variabel menggunakan kata kunci `var`, `let`, atau `const` diikuti dengan nama variabel. Contohnya:
```javascript
var nama = "John"; // var bisa digunakan di seluruh fungsi
let umur = 30; // let hanya berlaku di dalam blok tertentu
const PI = 3.14; // const merupakan nilai yang tidak dapat diubah
```

### Tipe Data:
- **String:** Data teks diapit oleh tanda kutip ganda atau tunggal.
    ```javascript
    let pesan = "Halo, dunia!";
    let kataKunci = 'JavaScript';
    ```

- **Number:** Merepresentasikan nilai numerik, baik bilangan bulat maupun desimal.
    ```javascript
    let umur = 25;
    let harga = 99.99;
    ```

- **Boolean:** Hanya memiliki dua nilai, yaitu `true` atau `false`.
    ```javascript
    let benar = true;
    let salah = false;
    ```

- **Array:** Kumpulan nilai yang terurut dalam sebuah variabel.
    ```javascript
    let angka = [1, 2, 3, 4, 5];
    let namaHari = ['Senin', 'Selasa', 'Rabu', 'Kamis', 'Jumat', 'Sabtu', 'Minggu'];
    ```

- **Objek:** Kumpulan pasangan properti-nilai.
    ```javascript
    let orang = {
        nama: "John",
        umur: 30,
        pekerjaan: "Developer"
    };
    ```
